### PR TITLE
Add aliases for kustomization application

### PR DESCRIPTION
**Pull request by mhaligowski**
https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#apply

## Standards checklist:

- [x] The PR title is descriptive.
- [x] The PR doesn't replicate another PR which is already open.
- [x] I have read the contribution guide and followed all the instructions.
- [x] The code follows the code style guide detailed in the wiki.
- [x] The code is mine or it's from somewhere with an MIT-compatible license.
- [x] The code is efficient, to the best of my ability, and does not waste computer resources.
- [x] The code is stable and I have tested it myself, to the best of my abilities.

## Changes:

- Adds aliases for applying kustomization directories

### CODE REVIEW

**Comment by mhaligowski** [comment]
Thanks @bartekpacia is there anything I need to do before merging? GH says I need write permission.

**Comment by bartekpacia** [comment]
Honestly, I'm not sure so I'll ask @larson-carter about this

**Comment by mcornella** [comment]
Please add the aliases to the plugin README before a merge can happen.

**Review by 53jk1 — APPROVED** [review]

**Comment by sylwit** [comment]
@mhaligowski I know it's been a while but any chance you fix the README so this PR can be merged. Thank you

File: plugins/kubectl/kubectl.plugin.zsh [hunk 1]
```diff
@@ -19,6 +19,9 @@ alias kca='_kca(){ kubectl "$@" --all-namespaces;  unset -f _kca; }; _kca'
 # Apply a YML file
 alias kaf='kubectl apply -f'
 
+# Apply a kustomization
+alias kak='kubectl apply -k'
```
Comment (carlosala) [inline plugins/kubectl/kubectl.plugin.zsh:23]: `kak` is the command for [`kakoune` editor](http://kakoune.org/), we'd need another name.

**Review by carlosala — CHANGES_REQUESTED** [review]

**Comment by carlosala** [comment]
This PR has been open for a while with important changes requested. Closing for the moment, feel free for anyone to reopen it.
Thanks for your contribution!

**Pull request closed without merging**